join.html

```diff
@@ -70,24 +91,105 @@ <h2 class="title">Curriculum</h2>
+        </div>
+        <div class="col no-gutter">
+            <p>
+                Our special events team hosts lots of <b>stand-alone</b> events and <b>short courses</b>, usually involving <b>collabs with other orgs</b>. We aim to <b>introduce CS</b> to kids and show them what it’s all about! <a class="tla-link" href="{{site.baseurl}}/citylab-cyber">Here</a> is an example of a past event that we’ve had. Interested? Here’s what we do: 
+            </p>
+            <ul>
+                <li>Collab with other clubs and work together to bring CS to kids (logistics+teaching)</li>
+                <li>Make curriculum and bring ideas to the table!</li>
+                <li><b>Flexible schedule and commitment</b>--you can choose what events you’d like to help with and how you help</li>
+            </ul>
+            <a class="button flex-center" href="{{site.general_interest}}"><span class="fa fa-file-alt"></span> General interest Form</a>
+        </div>
+    </div>
+
+    <h2 class="title">Speaker Series</h2>
+    <div class="row">
+        <div class="col no-gutter">
+            <p>
+                For our Speaker Series, we find one-off topics that we talk about with a class to get them interested in computer science and technology. If you want to meet new students and have fun discussions with them about tech, fill out the interest form to get involved! Our current Speaker Series include:
+            </p>
+            <ul>
+                <li><a class="tla-link" href="{{site.baseurl}}/ai-ethics">AI Ethics</a>: discuss ethical issues that might come about from machine learning and artificial intelligence</li>
+                <li>CS Pathways: talk about different career paths that you can pursue with CS knowledge</li>
+            </ul>
+            <a class="button flex-center" href="{{site.general_interest}}"><span class="fa fa-file-alt"></span> General interest Form</a>
+        </div>
+        <div class="col">
+            <img class="img-responsive" src="{{site.baseurl}}/img/undraw/join-speaker.svg" alt="Woman giving a presentation"/>
+        </div>
+    </div>
+
+    <h2 class="text-25x">Check out some of the great work our members are doing!</h2>
+    <div class="row">
+        <div class="col">
+            <a class="button flex-center" href="{{site.baseurl}}/files/Booleans_and_Conditionals_Emerson_Week_4.pdf"><span class="fa fa-file-alt"></span>Emerson (Python I) Booleans and Conditionals PPT</a>
+        </div>
+        <div class="col no-gutter">
+            <a class="button flex-center" href="{{site.baseurl}}/files/Worksheet_Data_Types_Operators.pdf"><span class="fa fa-file-alt"></span>Emerson (Python I) Data Types Worksheet</a>
+        </div>
+        <div class="col no-gutter">
+            <a class="button flex-center" href="{{site.baseurl}}/dev#projects"><span class="fa fa-file-alt"></span>Dev Team Projects</a>
+        </div>
```

Comment:
I wouldn't use the `.row` and `.col` classes to do this kind of layout - CSS grid might be better suited!

Also, these buttons are really big / a bit hard to read - it may be better to have a general card-like component that describes the thing, and then just a button that links to the example.